packages/cadl-ranch-specs/http/type/model/inheritance/non-discriminator/main.tsp

```diff
@@ -0,0 +1,53 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@doc("Illustrates non-discriminator inheritance model.")
+@scenarioService("/type/model/inheritance/non-discriminator")
+namespace Type.Model.Inheritance.NonDiscriminator;
```

Comment:
```suggestion
namespace Type.Model.Inheritance.NoDiscriminator;
```

or `NotDiscriminated` I think would be better naming.

Reply:
rename the whole scenario `not-discriminated`